requirements.txt

```diff
@@ -43,9 +43,14 @@ nocasedict>=1.0.1
 # urllib3 1.25.9 addressed issue 38834 reported by safety
 # urllib3 needs to be pinned to <1.25 for requests <2.22.0
 # urllib3 1.26.5 vendors six 1.16.0 which is needed on Python 3.10 to remove ImportWarning
-urllib3>=1.25.9; python_version == '2.7'
-urllib3>=1.25.9; python_version >= '3.5' and python_version <= '3.9'
-urllib3>=1.26.5; python_version >= '3.10'
+# urllib3 2.0 incompatible with python 2.7, 2.5, 3.6
+urllib3>=1.25.9,<2.0.0; python_version == '2.7'
+urllib3>=1.25.9,<2.0.0; python_version >= '3.5' and python_version <= '3.6'
```

Comment:
Given that urllib3 2.0 requires py>=3.7, it does not need to be pinned for py<=3.6

Reply:
DONE.